pkg/utils/tekton/controller.go

```diff
@@ -25,9 +26,32 @@ type KubeController struct {
 	Namespace  string
 }
 
+type Bundles struct {
+	BuildTemplatesBundle            string
+	HACBSTemplatesBundle            string
+	HACBSCoreServiceTemplatesBundle string
+}
+
+func newBundles(kube *kubeCl.K8sClient) (*Bundles, error) {
+	buildPipelineDefaults, err := kube.KubeInterface().CoreV1().ConfigMaps("build-templates").Get(context.TODO(), "build-pipelines-defaults", metav1.GetOptions{})
+	if err != nil {
+		return nil, err
+	}
+
+	bundle := buildPipelineDefaults.Data["default_build_bundle"]
+
+	return &Bundles{
+		BuildTemplatesBundle:            bundle,
+		HACBSTemplatesBundle:            strings.Replace(bundle, "/build-", "/hacbs-", 1),
+		HACBSCoreServiceTemplatesBundle: strings.Replace(bundle, "/build-", "/hacbs-core-service-", 1),
```

Comment:
On my cluster `bundle` is `quay.io/redhat-appstudio/build-templates-bundle:861e28f4eb2380fd1531ee30a9e74fb6ce496b9f`

`strings.Replace(bundle, "/build-", "/hacbs-", 1)` gives me the following: `quay.io/redhat-appstudio/hacbs-templates-bundle:861e28f4eb2380fd1531ee30a9e74fb6ce496b9f`, which is valid.

But `strings.Replace(bundle, "/build-", "/hacbs-core-service-", 1)` gives me a pull spec that returns "manifest unknown": `quay.io/redhat-appstudio/hacbs-core-service-templates-bundle:861e28f4eb2380fd1531ee30a9e74fb6ce496b9f`

Reply:
Thanks! Correct, quay.io/redhat-appstudio/hacbs-core-service-templates-bundle seems to exist only with `latest`, which is not very excellent. I'll change it, I was looking at the output of `hack/build-and-push.sh` in build-definitions and failed to notice that there is no git id based tag for it.